pulpcore/pulpcore/app/viewsets/base.py

```diff
@@ -14,6 +14,9 @@
 from rest_framework.generics import get_object_or_404
 from rest_framework.schemas import AutoSchema
 
+NAME_FILTER_OPTIONS = ['exact', 'in']
+DATETIME_FILTER_OPTIONS = ['lt', 'lte', 'gt', 'gte', 'range']
```

Comment:
Other options:  "day", "week", "month", "day__range", "minute__range", "week__range"...

I had to restrain myself from going crazy after I figured out that that shit fuckin' works!

We can totally do a query like ```?created__hour__range=14,17``` if we want to enable that.

Reply:
Could you add some comments with example usage?

Reply:
+1